content/docs/capabilities/kubernetes-access.mdx

```diff
@@ -0,0 +1,241 @@
+---
+# cSpell:ignore kubeconfig mycluster
+
+title: Kubernetes `kubectl` Integration
+sidebar_label: Kubernetes Access
+description: This article describes Pomerium's integration with the Kubernetes API Server
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+
+# Kubernetes Access
+
+## Background
+
+Kubernetes supports [a variety](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#authentication-strategies) of ways to perform authentication against the API Server. While there is tremendous flexibility in the core product, operators can encounter various practical challenges:
+
+- Cloud providers typically support only their native IAM implementation, which may not integrate with your IdP
+- OIDC providers may not provide group claims, requiring manual mappings to RBAC roles
+- Your IdP may not be reachable by the kubernetes control plane
+- Access is managed per cluster without central control
+- Dynamic privilege escalation during incidents are slow or cumbersome RBAC changes
+- VPN based protection may not be possible or desirable
+
+Similarly, Kubernetes supports native [audit logging](https://kubernetes.io/docs/tasks/debug-application-cluster/audit/) capabilities, but can also run into practical challenges:
+
+- Cloud provider deployments may be ecosystem locked with limited tooling, if any
+- Cross-cluster and cross-service audit trails must be stitched together by the operator
+
+## Solution
+
+Pomerium can be leveraged as a proxy for user requests to the API Server.
+
+- Any supported IdP can be supported for authentication, in any environment
+- Group membership is supported consistently
+- Centralized, dynamic, course grained access policy
+- Global, cross resource access and audit trail
+- API server protection without the operational challenges of VPN
+- Can be hosted inside your kubernetes cluster!
+
+## How it works
+
+Pomerium supports two different methods of authenticating to the Kubernetes API server:
+
+- **Impersonation Headers**
+- **Pomerium JWT** using Structured Authentication (Kubernetes 1.30+)
```

Comment:
Can we clarify the pros and cons to help the user choose the tab that is better for them?

e.g.
- **Impersonation Headers** works on all managed or unmanaged kubernetes clusters, but provides broad service account access
- **Pomerium JWT** relies on Structured Authentication Configuration, available in Kubernetes 1.30+ and may depend on your managed cluster provider's feature availability, but provides a more secure authorization policy

Probably you could clean those two statements up better than this.

Reply:
Also, let's stay consistent and call this Structured Authentication Configuration rather than just Structured Authentication.